src/methods/pivot_point_standard.rs

```diff
@@ -0,0 +1,134 @@
+//! The Pivot Point indicator helps traders and analysts identify potential support and resistance levels for a given
+//! trading day or time period. Pivot Points are particularly popular in day trading and short-term trading strategies.
+//!
+
+#[cfg(feature = "serde")]
+use serde::{Deserialize, Serialize};
+
+use crate::core::{Method, ValueType, OHLCV};
+
+#[derive(Debug, Clone, Copy)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+/// The Pivot Point indicator is calculated based on the high, low, and close prices of the previous trading
+/// period (such as a day, week, or month) and provides several support and resistance levels.
+///
+/// ## Links
+///
+/// * <https://en.wikipedia.org/wiki/Pivot_point_(technical_analysis)>
+///
+pub struct PivotPointStandard {
+	close: ValueType,
```

Comment:
Do we even need internal state? It seems like `PivotPointStandard` may be simple unit struct without internal state.